### DISCUSSION TITLE
File uYouPlus v17.48.2_2.1.ipa is infected with Trojan virus:Script/Wacatac.H!ml

### DISCUSSION THREAD

**Discussion by nvandoan** [qnblackcat/uYouPlus#758] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/120364620/206986646-c959b1f7-705f-4567-bfe1-e145e5f44074.png)

**Comment by greendoescode** [comment, marked as answer]
Windows defender loves calling anything a virus
